src/Umbraco.New.Cms.Infrastructure/Umbraco.New.Cms.Infrastructure.csproj

```diff
@@ -10,4 +10,8 @@
     <ProjectReference Include="..\Umbraco.New.Cms.Core\Umbraco.New.Cms.Core.csproj" />
     <ProjectReference Include="..\Umbraco.Infrastructure\Umbraco.Infrastructure.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenIddict.Abstractions" Version="3.1.1" />
```

Comment:
@kjac is this really necessary? Double check and clean up...

Reply:
It is indeed necessary... the `OpenIddictCleanup` is located in this project, and it references the OpenIddict abstractions.